mne/utils/docs.py

```diff
@@ -144,6 +144,18 @@
 docdict['reject_by_annotation_raw'] = docdict['reject_by_annotation_all'] + """
     Has no effect if ``inst`` is not a :class:`mne.io.Raw` object.
 """
+docdict['annot_ch_names'] = """
+ch_names : array-like, shape (n_annotations,) | None
+    1D object array of tuples of channel names associated with the annotations.
```

Comment:
```suggestion
    1D array of tuples of channel names associated with the annotations.
```

Reply:
I think "object array" is more specific and hence useful here, like saying "float array"

Reply:
Ah, I didn't know you were referring to a string array. Object is a very generic term. I know numpy and pandas use these to refer to arrays that contain objects, but mostly these are strings. Therefore, I'd use string array here.

Reply:
Even more precise would be list of list of strings (where list could be replaced by array).

Reply:
`Sequence[Union[None, Sequence[str]]]` would be the correct type hint I guess?

Reply:
Or maybe even `Sequence[Optional[Sequence[str]]]`...

Reply:
> I know numpy and pandas use these to refer to arrays that contain objects, but mostly these are strings

It's not a string array, it's an object array -- they are different. An ndarray with dtype `str` gets its values stored in memory by NumPy. An ndarray with dtype object just holds pointers to some other Python objects. I'm used to these being called "object array"s but I'll reword things to make this hopefully clearer.

Reply:
Yes, that's very numpy-specific. A sequence of string sequences would be accurate and easy to understand.

Reply:
List is also OK!